### PR TITLE
536 change tree icon default expanded

### PR DESCRIPTION
**Pull request by wouter1975**
- Change tree icon of root PB
- Use title for root PB in tree instead of PB name
- At page load the root PB is expanded in card view

### CODE REVIEW

File: packages/orchestrator-ui-components/src/components/WfoTree/WfoTreeNode.tsx [hunk 1]
```diff
@@ -44,10 +44,7 @@ export const WfoTreeNode: FC<WfoTreeNodeProps> = ({
     const expanded = expandedIds.includes(item.id);
     const selected = selectedIds.includes(item.id);
 
-    let expandIcon = expanded ? 'arrowDown' : 'arrowRight';
-    if (item.id === 0) {
```
Comment (DutchBen) [inline packages/orchestrator-ui-components/src/components/WfoTree/WfoTreeNode.tsx:48]: What did this condition do and are we sure you don't need it anymore? 
It seems it toggles the exapandIcon prop only for the first item which seems significant

**Review by DutchBen — APPROVED** [review]

File: packages/orchestrator-ui-components/src/components/WfoSubscription/WfoRelatedSubscriptions.tsx [hunk 2]
```diff
@@ -86,7 +86,10 @@ export const WfoRelatedSubscriptions = ({
             field: 'description',
             name: t('description'),
             render: (value, record) => (
-                <Link href={`/subscriptions/${record.subscriptionId}`}>
+                <Link
+                    target="_blank"
+                    href={`/subscriptions/${record.subscriptionId}`}
```
Comment (ricardovdheijden) [inline packages/orchestrator-ui-components/src/components/WfoSubscription/WfoRelatedSubscriptions.tsx:91]: We hava a constants file with all app paths:
`packages/orchestrator-ui-components/src/components/WfoPageTemplate/paths.ts`

Also I see we did not always use that in the project, just een FYI

**Review by ricardovdheijden — APPROVED** [review]

File: packages/orchestrator-ui-components/src/components/WfoSubscription/WfoSubscriptionProductBlock.tsx [hunk 3]
```diff
@@ -84,7 +84,7 @@ export const WfoSubscriptionProductBlock = ({
                                 )}
                             </h3>
                         </EuiText>
-                        <EuiText>{`${t(
+                        <EuiText size={'s'}>{`${t(
```
[the remark this answers is not in the document]
Comment (wouter1975) [reply packages/orchestrator-ui-components/src/components/WfoSubscription/WfoSubscriptionProductBlock.tsx:87]: Ok, I will change that throughout the project

File: packages/orchestrator-ui-components/src/components/WfoSubscription/WfoRelatedSubscriptions.tsx (hunk 2 above)
Comment (wouter1975) [reply packages/orchestrator-ui-components/src/components/WfoSubscription/WfoRelatedSubscriptions.tsx:91]: ok, i will leave it as is.

**Pull request merged**